rule.php

```diff
@@ -84,14 +84,15 @@ public static function make (quiz $quizobj, $timenow, $canignoretimelimits) {
      * @throws coding_exception
      */
     public static function add_settings_form_fields(mod_quiz_mod_form $quizform, MoodleQuickForm $mform) {
+        global $DB;
         $defaults = settings_provider::get_quiz_defaults();
         $hideifs = settings_provider::get_quiz_hideifs();
 
         // Insert all the form elements before the 'security' section as a group.
         foreach (settings_provider::get_quiz_element_types() as $name => $type) {
 
             // Check if the user has capability to edit setting, otherwise use hidden setting type.
-            if (!has_capability('quizaccess/seb:manage_' . $name, $quizform->get_context())) {
+            if ($type != 'header' && !has_capability('quizaccess/seb:manage_' . $name, $quizform->get_context())) {
```

Comment:
Ah, nice. I also saw this was required from my previous accepted approval. No action required since you've fixed it up!